pyphare/pyphare/simulator/simulator.py

```diff
@@ -170,12 +171,16 @@ def times(self):
             self.timeStep(),
         )
 
-    def run(self, plot_times=False):
+    def run(self, plot_times=False, monitoring=True):
```

Comment:
maybe default to False, anything else besides that @nicolasaunai ?

this PR has a fix for scope timings that would be nice to have generally